### PR TITLE
events explorer

### PR DESCRIPTION
**Pull request by cswatt**
### Preview

https://docs-staging.datadoghq.com/cswatt/events-explorer/events

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by condrenjustin** [comment]
Here is some feedback for the updated events explorer and monitor documentation

https://docs.google.com/document/d/1KuP-He_7fe123uZf3XYjWXkEJX5Ixs9sVfASUfJXp3c/edit?usp=sharing

**Review by apigirl — APPROVED** [review]
This looks great! I know this isn't changes you made, but host name should be hostname in a couple of places

File: content/en/events/_index.md [hunk 1]
```diff
@@ -78,11 +127,20 @@ sources:nagios,chef status:error cassandra
...
+{{< /site-region >}}
+{{< site-region region="eu" >}}
 ### Aggregation
 
 By default, related events are aggregated when displayed in the events stream. To show unaggregated events, un-check the **Aggregate related events** box at the top right of your event stream:
 
 {{< img src="events/event_stream_aggregated.png" alt="Aggregated event stream"  style="width:50%;" >}}
+{{< /site-region >}}
 
 ### Notifications
```
Comment (rayene290) [inline content/en/events/_index.md:145]: I thought these weren't supported in GovCloud? The Event Management team can confirm though.

File: content/en/monitors/monitor_types/event.md [hunk 2]
```diff
@@ -37,6 +38,43 @@ Choose your alert grouping:
...
+{{< /site-region >}}
+{{< site-region region="gov" >}}
+
+### Define the search query
+
+As you define the search query, the top graph updates.
+
+1. Construct a search query using the same logic as a [log explorer search][1].
+2. Choose to monitor over an event count, facet, or measure:
+    * **Monitor over a log count**: Use the search bar (optional) and do **not** select a facet or measure. Datadog evaluates the number of events over a selected time frame, then compares it to the threshold conditions.
```
Comment (condrenjustin) [inline content/en/monitors/monitor_types/event.md:68]: This should read **Monitor over an event count** rather than log count

**Comment by apigirl** [comment]
closing this because it got moved into https://github.com/DataDog/documentation/pull/9574

**Pull request closed without merging**